### PR TITLE
Add es2015 example of loading highcharts + module

### PR DESCRIPTION
**Pull request by pgilad**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -34,3 +34,22 @@ var chart = new Highcharts.Chart({
 });
 ```
 
+Example of using Highcharts in `ES2015`:
+
+```js
+// load Highcharts
+import Highcharts from 'highcharts';
```
Comment (graingert) [inline readme.md]: this should be
```
import highcharts from 'highcharts/highcharts.src';
const Higcharts = typeof highcharts === 'function' ? highcharts(jsdomRoot()) : highcharts;
```

**Comment by hugomrdias** [comment]
why is the minified version the main in package.json or even in the npm build ? 

having to do ` import highcharts from 'highcharts/highcharts.src';` instead of `import highcharts from 'highcharts/highcharts';` has several down sides and isn't standard. we lose type definitions, we can't use the dependencies field of package.json for bundling dependencies, we lose sourcemaps  etc etc

File: readme.md [hunk 2]
```diff
@@ -34,3 +34,22 @@ var chart = new Highcharts.Chart({
...
+// load the exporting module
+import Exporting from 'highcharts/modules/exporting';
+
+// initialize exporting module
+Exporting(Highcharts);
+
+// Generate the chart
+const chart = new Highcharts.Chart({
```
Comment (pgilad) [inline readme.md]: should be `chart` lower case

**Comment by jon-a-nygaard** [comment]
Thanks for your contribution!

**Pull request merged**